.github/CONTRIBUTING.md

```diff
@@ -1,83 +1,64 @@
 # Contributing
-
-Before making any contributions, please sign one of the contributor
-license agreements below.
-
-Fork the repo, develop and test your code changes.
+Before making any contributions, please sign one of the contributor license agreements below. Fork the repo, develop and test your code changes.
 
 Install all dependencies including development requirements by running:
 
 ``` sh
-$ npm install -d
+$ npm install
 ```
 
-Tests are run using mocha. To run all tests just run:
+Builds or tests require the installation of [`make`](https://www.gnu.org/software/make/).  To perform the first build, go ahead and run it!
```

Comment:
...which is likely already installed if you develop on Linux or Mac.

Reply:
(otherwise the text might sound frightening for people too young to see `make` in action)